indico/modules/events/timetable/controllers/manage.py

```diff
@@ -283,6 +311,28 @@ def _get_references(self, data: list[dict]) -> list[ContributionReference]:
                                                     value=entry['value']))
         return references
 
+    def _move_entry(self, session_block_id):
+        entry = self.contrib.timetable_entry
+
+        self.contrib.session_block = None
+        self.contrib.session = None
```

Comment:
Shouldn't we only do this if there is no target block? If there is, it will already overwrite it in else. Now we just remove it by default